CMakeLists.txt

```diff
@@ -186,12 +191,17 @@ RELATIVE_PROTOBUF_GENERATE_CPP(PROTO_SRCS PROTO_HDRS ${ONNX_ROOT}
     onnx/onnx-operators.in.proto
 )
 
+file(GLOB_RECURSE onnx_INCLUDE_DIRS "${ONNX_ROOT}/onnx/*.h")
```

Comment:
wait I think this is finding a list of header files, not "dirs". We have set ONNX_INCLUDE_DIRS, will it be enough by just adding it to the include dirs when compiling the test cc files?